front/src/common/hooks/use-Auto-Scroll.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
It's better to name files with lowercase. In this case, I usually name this file like: `auto-scroll.hook.ts`